tests/test_jmcomic/test_jm_exception.py

```
@@ -0,0 +1,100 @@
+from test_jmcomic import *
```

Ruff (0.16.2)

[error] `from test_jmcomic import *` used; unable to detect undefined names

(F403)